### ISSUE TITLE
install.sh: Create an env var that lets the user provide their own admin token

### ISSUE THREAD

**Issue by paulproteus** [sandstorm-io/sandstorm#1028]
This paves the way for more-fully-automated installs.

Sample invocation:

```
sudo ADMIN_TOKEN_STRING=17dfb51a21cfbc2b61c2192fbe675f0357932eba bash install.sh
```

**Comment by paulproteus** [comment]
Self-assigning so I don't forget about it, but I don't have a specific timeframe for doing this yet. Feel free to steal it from me if someone else wants to.

**Comment by paulproteus** [comment]
Fixed in 6b5351c4a58c29e9bf3ad6d16bae27895aeaad38
